FileManager/src/Controller/Admin/AttachmentsController.php

```diff
@@ -241,9 +242,9 @@ public function admin_delete($id = null) {
  * @return void
  * @access public
  */
-	public function admin_browse() {
+	public function browse() {
 		$this->layout = 'admin_popup';
-		$this->admin_index();
+		$this->index();
```

Comment:
Would this be suitable for this?

http://book.cakephp.org/3.0/en/controllers.html#redirecting-to-another-action-on-the-same-controller